modules/simple-payments/paypal-express-checkout.js

```diff
@@ -106,10 +114,13 @@ var PaypalExpressCheckout = {
 		paypal.Button.render( {
 			env: env,
 			commit: true,
+
 			style: {
 				label: 'pay',
-				color: 'blue'
+				shape: 'pill',
```

Comment:
Just realised there is `rect` option for `shape`. I think I prefer the rectangle than the pill that feels a bit old school to me. Let's go with `rect`.